QLSettings.cpp

```diff
@@ -127,6 +132,7 @@ QLSettings::SaveSettings()
 	settings.AddInt32("singleclick", fSingleClick);
 	settings.AddInt32("ontop", fOnTop);
 	settings.AddInt32("show ignore", fShowIgnore);
+	settings.AddInt32("searchstart", fSearchStart);
```

Comment:
Please move in front of the "single click" setting, to reflect the order in the GUI.